src/core/observables/CylindricalDensityProfile.hpp

```diff
@@ -41,7 +41,8 @@ class CylindricalDensityProfile : public CylindricalPidProfileObservable {
 
     for (auto p : particles) {
       histogram.update(Utils::transform_coordinate_cartesian_to_cylinder(
-          folded_position(traits.position(p), box_geo) - center, axis));
+          folded_position(traits.position(p), box_geo) - trafo_params->center(),
+          trafo_params->axis(), trafo_params->orientation()));
```

Comment:
since you wanted to make sure we all agree on the naming... I think it would not harm the readability to use the same parameter name as in the python interface 😁 "trafo" is also a german abbreviation IIRC